libs/watt/package/text-field/watt-text-field.component.ts

```diff
@@ -158,6 +166,14 @@ export class WattTextFieldComponent implements ControlValueAccessor, AfterViewIn
       this.inputField().nativeElement.value = trimmedValue;
       this.formControl().setValue(trimmedValue);
     });
+
+    // Programmatically focus the input if autoFocus is true
+    if (this.autoFocus()) {
+      // Use requestAnimationFrame to ensure the element is ready in the DOM
+      requestAnimationFrame(() => {
+        this.inputField().nativeElement.focus();
+      });
+    }
```

Comment:
suggestion: this is a nice feature, but it is a shame that it is only available on the watt-text-field - consider moving this logic to watt-field, so that all fields can benefit from it.